src/poprox_concepts/domain/newsletter.py

```diff
@@ -11,6 +12,7 @@ class Impression(BaseModel):
     position: int
     article: Article
     created_at: datetime | None = None
+    experiment_extra: dict[str, Any] | None = None
```

Comment:
In principal this looks fine, but it makes me wonder how we plan on storing this, and if there's a way to say "json-compatible-data" instead of `dict[str, Any] | None`

Reply:
Follow-up to self. A quick google makes it look like `dict[str, Any] | None` is the normal way to do this. 

If this is the type we want I imagine we're just going to store it as a json field instead of doing it manually?